tests/solvers/test_radau5.py

```diff
@@ -1713,12 +1830,13 @@ def test_nmax_steps(self):
         This tests the error upon exceeding a set maximum number of steps
         """
         sim = Radau5DAE(self.mod)
-        sim.solver = 'c'
 
         sim.maxh = 1.e-1
         sim.maxsteps = 9
 
-        nose.tools.assert_raises(Radau5Error, sim.simulate, 1.)
+        err_msg = f"The solver took max internal steps but could not reach the next output time."
```

Comment:
You are using an f-string here but not formatting the string, I am just adding the comment here in case you missed something.